docs/eks-installation.md

```diff
@@ -11,22 +11,19 @@ This would restart existing PODs (except in host network) so that Antrea can als
 kubectl apply -f https://raw.githubusercontent.com/vmware-tanzu/antrea/master/build/yamls/antrea-eks-node-init.yml
```

Comment:
Not introduced by you but the sentence above:
> This would restart existing PODs (except in host network) so that Antrea can also manage it, once installed.

Is not great... Can you use this instead:
> This will restart existing Pods (except those in host network) so that Antrea can also manage them (i.e. enforce NetworkPolicies on them), once it is installed.

Reply:
Done.